vault/dendron.tutorial.linking-notes.md

```diff
@@ -2,43 +2,59 @@
 id: 9MZBqhrijEM4QpZRa5t08
 title: Linking Notes
 desc: Linking Notes
-updated: 1637551418396
+updated: 1639428297065
 created: 1625563999532
 nav_order: 2
 ---
 
 ### Links
 
-Dendron supports links between your notes, which can help you organically connect your thoughts and build up a powerful and **personal** knowledge graph.
+Dendron supports links between your notes, which can help you organically connect your thoughts and build up a powerful and personal knowledge graph.
 
-> 🌱 To create a link, just start typing [[ and Dendron will show you a list of notes in your workspace that you can link to. Try it in the editor pane now.
+> 🌱 **ACTION:** To create a link, start typing `[[` and Dendron will show you a list of notes in your workspace that you can link to. Try it in the editor view pane now.
 
-Example Text: `[[dendron.tutorial]]`
-Resulting Link: [[dendron.tutorial]]
+- Example Text: `[[dendron.tutorial]]`
+- Resulting Link: [[dendron.tutorial]]
 
-To navigate to the note in the link, you can just click on the link in the preview pane. In the editor pane, you can move the cursor into the link and hit `Ctrl+Enter`.
+To navigate to the note in the link, click the link in the preview pane. In the editor pane, move the cursor into the link and hit `Ctrl+Enter`.
 
-You can switch back to the previous note with the tab switching shortcut (`Ctrl+Tab` on most systems).
+You can switch back to the previous note with the tab switching shortcut: `Ctrl+Tab` on most systems.
 
 #### Backlinks
 
-Take a look at the backlinks panel on the bottom left section of your sidebar. The backlinks panel shows you all notes that have linked to the current note. This is useful for helping to establish context.
+![[dendron.tutorial.user-interface#7-backlinks-pane,1:#*]]
```

Comment:
since we already talk about this, not great to exactly copy what we've talked about. maybe a wiki link inside a TIP instead of a note reference here?